servers/navigation/navigation_globals.h

```diff
@@ -53,13 +52,12 @@ namespace NavigationDefaults2D {
 // Rasterization.
 
 // Same as in 3D but larger since 1px is treated as 1m.
-constexpr float navmesh_cell_size{ 1.0f }; // Must match ProjectSettings default 2D cell_size.
-constexpr float navmesh_cell_size_min{ 0.01f };
-constexpr auto navmesh_cell_size_hint{ "0.001,100,0.001,or_greater" };
+constexpr float NAV_POLYGON_CELL_SIZE = 1.0f; // Must match ProjectSettings default 2D cell_size.
```

Comment:
This should be kept navmesh. We try to have 2d and 3d named navmesh / navigation mesh where possible to simplify documentation and code sharing and because users are always confused when you mix them.